scripts/msmarco_v2/build_psg_doc_idmap.py

```diff
@@ -0,0 +1,63 @@
+#
```

Comment:
Rename `build_passage_to_doc_id_map.py`?

Let's be explicit? No need to save a few characters since we have tab complete.

And `id_map` is more readable.